app/models/assignment.rb

```diff
@@ -37,13 +38,19 @@ class Assignment < ActiveRecord::Base
 
   scope :excluding_incomplete_features_for, ->(app_build) do
     joins(:split).where(
-      Arel::Nodes::Or.new(
-        arel_table[:force].eq(true),
-        Split.arel_excluding_incomplete_features_for(app_build)
+      Arel::Nodes::Grouping.new(
+        Arel::Nodes::Or.new(
+          arel_table[:force].eq(true),
+          Split.arel_excluding_incomplete_features_for(app_build)
+        )
       )
     )
   end
 
+  scope :excluding_remote_kills_for, ->(app_build) do
+    joins(:split).merge(Split.excluding_remote_kills_for(app_build))
+  end
```

Comment:
This feels very normal for how fancy the generated SQL is.

Reply:
Oh this should respect force: true also. Nothing should prevent a determined tester from seeing a broken variant.

Reply:
Ok, that ended up being tricky and adding a whole nother level of expressiveness to `AppRemoteKill.affecting`. Will do another review to outline that behavior.